src/Install/Prerequisite/PhpVersion.php

```diff
@@ -20,13 +22,15 @@ public function __construct($minVersion)
         $this->minVersion = $minVersion;
     }
 
-    public function check()
+    public function problems(): Collection
     {
         if (version_compare(PHP_VERSION, $this->minVersion, '<')) {
-            $this->errors[] = [
+            return collect()->push([
                 'message' => "PHP $this->minVersion is required.",
                 'detail' => 'You are running version '.PHP_VERSION.'. Talk to your hosting provider about upgrading to the latest PHP version.',
```

Comment:
Most commonly the suggestion is to talk to the "system administrator", referring to a hosting provider might be too opinionated?

Reply:
Thanks, good suggestion.